src/stdlib.dt

```diff
@@ -156,3 +156,9 @@
 [ [ cond msg ]:   [ msg epl   1 exit ] cond not do? ]
 "( <cond> <message> -- ) Naive assertion that requires a condition to be truthy. If falsy, it will print a message to standard error and fail with an exit code of 1."
 \assert-true define
+
+### REPL Helpers ###
+
+[ defs [ dup usage swap green p norm "\t" p pl ] each ]
```

Comment:
Makes sense to me! I might rename the binding to make it more obvious that it's not using the builtin word, but otherwise I think it makes it more clean.

Reply:
Sounds good to me. `def` will be pretty ubiquitous